data/coalition outfits.txt

```diff
@@ -143,6 +143,21 @@ outfit "Cooling Module"
 	"cooling energy" 1.4
 	description "This cooling module consumes energy. To conserve power, it does not ramp up to full strength unless a ship is very close to overheating."
 
+outfit "Scanning Module"
+	category "Systems"
+	licenses
+		Heliarch
+	cost 453600
+	thumbnail "outfit/scan module"
+	"mass" 3
+	"outfit space" -3
+	"outfit scan power" 25
+	"outfit scan speed" 1
+	"cargo scan power" 25
+	"cargo scan speed" 1
+	"tactical scan power" 25
+	description "This system allows the Heliarchs to scan nearby ships."
```

Comment:
This description could definitely be better. How about something like `All Heliarch ships come standard with an array of scanning modules used to ensure that Coalition citizens are not transporting any illegal goods in their ships, such as weapons and ammunition.`

Reply:
The description ought to be about the scanning module, not about Heliarch ships.

Reply:
The tactical scanner's description: `"A tactical scanner allows your ship to determine the fuel, energy, heat, and crew complement of nearby targets. For centuries tactical scanners were only employed by the Navy, but recently Kraz Cybernetics has begun manufacturing them for use by militia ships."`

So talking about the ships that the scanner is on is alright, as long as the abilities of the scanner are also mentioned. And to expand my description to also hit on the tactical scan aspect: `All Heliarch ships come standard with an array of scanning modules used to ensure that Coalition citizens are not transporting any illegal goods in their ships, such as weapons and ammunition, as well as find out the status of the ship being targeted.`

Reply:
Mentioning ships is fine, but the emphasis ought to be on the outfit. Compare *TC [function]. TC user, new TC manufacturer.* with: *Heliarch ships' default equipment, Heliarch reason, Heliarch targetting.*

Reply:
Whatever. A better description can be made when it's available for purchase or simply in outfitters.